sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -16,7 +16,7 @@
       netcoreapp2.1 is out of support but we'll keep is as a target for a bit
       to give customers time to migrate without losing perf optimizations
      -->
-    <NoWarn>$(NoWarn);NETSDK1138</NoWarn>
+    <NoWarn>$(NoWarn);NETSDK1138;NU1902;NU1903</NoWarn>
```

Comment:
I don't see those coming through locally, what warnings did you need to suppress?

Reply:
Seems that this is a new change in NET8
https://learn.microsoft.com/en-us/dotnet/core/compatibility/sdk/8.0/dotnet-restore-audit#new-behavior

Without those suppressions, I get the following errors:

```
  Determining projects to restore...
C:\REPOS\AzureSDK\sdk\core\Azure.Core\src\Azure.Core.csproj : error NU1903: Warning As Error: Package 'Microsoft.NETCore.App' 2.1.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-2xjx-v99w-gqf3 [C:\REPOS\AzureSDK\sdk\monitor\Azure.Monitor.OpenTelemetry.Exporter\tests\Azure.Monitor.OpenTelemetry.Exporter.AotCompatibilityTestApp\Azure.Monitor.OpenTelemetry.Exporter.AotCompatibilityTestApp.csp
roj]
C:\REPOS\AzureSDK\sdk\core\Azure.Core\src\Azure.Core.csproj : error NU1902: Warning As Error: Package 'Microsoft.NETCore.App' 2.1.0 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-3gp9-h8hw-pxpw [C:\REPOS\AzureSDK\sdk\monitor\Azure.Monitor.OpenTelemetry.Exporter\tests\Azure.Monitor.OpenTelemetry.Exporter.AotCompatibilityTestApp\Azure.Monitor.OpenTelemetry.Exporter.AotCompatibilityTestApp
.csproj]
C:\REPOS\AzureSDK\sdk\core\Azure.Core\src\Azure.Core.csproj : error NU1903: Warning As Error: Package 'Microsoft.NETCore.App' 2.1.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-3w5p-jhp5-c29q [C:\REPOS\AzureSDK\sdk\monitor\Azure.Monitor.OpenTelemetry.Exporter\tests\Azure.Monitor.OpenTelemetry.Exporter.AotCompatibilityTestApp\Azure.Monitor.OpenTelemetry.Exporter.AotCompatibilityTestApp.csp
roj]
C:\REPOS\AzureSDK\sdk\core\Azure.Core\src\Azure.Core.csproj : error NU1902: Warning As Error: Package 'Microsoft.NETCore.App' 2.1.0 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-5633-f33j-c6f7 [C:\REPOS\AzureSDK\sdk\monitor\Azure.Monitor.OpenTelemetry.Exporter\tests\Azure.Monitor.OpenTelemetry.Exporter.AotCompatibilityTestApp\Azure.Monitor.OpenTelemetry.Exporter.AotCompatibilityTestApp
.csproj]
C:\REPOS\AzureSDK\sdk\core\Azure.Core\src\Azure.Core.csproj : error NU1903: Warning As Error: Package 'Microsoft.NETCore.App' 2.1.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-6px8-22w5-w334 [C:\REPOS\AzureSDK\sdk\monitor\Azure.Monitor.OpenTelemetry.Exporter\tests\Azure.Monitor.OpenTelemetry.Exporter.AotCompatibilityTestApp\Azure.Monitor.OpenTelemetry.Exporter.AotCompatibilityTestApp.csp
roj]
C:\REPOS\AzureSDK\sdk\core\Azure.Core\src\Azure.Core.csproj : error NU1903: Warning As Error: Package 'Microsoft.NETCore.App' 2.1.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-g5vf-38cp-4px9 [C:\REPOS\AzureSDK\sdk\monitor\Azure.Monitor.OpenTelemetry.Exporter\tests\Azure.Monitor.OpenTelemetry.Exporter.AotCompatibilityTestApp\Azure.Monitor.OpenTelemetry.Exporter.AotCompatibilityTestApp.csp
roj]
C:\REPOS\AzureSDK\sdk\core\Azure.Core\src\Azure.Core.csproj : error NU1902: Warning As Error: Package 'Microsoft.NETCore.App' 2.1.0 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-vgwq-hfqc-58wv [C:\REPOS\AzureSDK\sdk\monitor\Azure.Monitor.OpenTelemetry.Exporter\tests\Azure.Monitor.OpenTelemetry.Exporter.AotCompatibilityTestApp\Azure.Monitor.OpenTelemetry.Exporter.AotCompatibilityTestApp
.csproj]
C:\REPOS\AzureSDK\sdk\core\Azure.Core\src\Azure.Core.csproj : error NU1902: Warning As Error: Package 'Microsoft.NETCore.App' 2.1.0 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-x5qj-9vmx-7g6g [C:\REPOS\AzureSDK\sdk\monitor\Azure.Monitor.OpenTelemetry.Exporter\tests\Azure.Monitor.OpenTelemetry.Exporter.AotCompatibilityTestApp\Azure.Monitor.OpenTelemetry.Exporter.AotCompatibilityTestApp
.csproj]
  Restored C:\REPOS\AzureSDK\sdk\monitor\Azure.Monitor.OpenTelemetry.Exporter\src\Azure.Monitor.OpenTelemetry.Exporter.csproj (in 680 ms).
  Failed to restore C:\REPOS\AzureSDK\sdk\core\Azure.Core\src\Azure.Core.csproj (in 684 ms).
```

I'm sure someone will resolve this given time. I needed to suppress it locally so I could view the AOT/Trimmer warnings.

Reply:
It looks like this is coming from the netcoreapp2.1 target. I'm going to open an issue about this.